packages/components/tags/tag.component.ts

```diff
@@ -603,12 +592,25 @@ export class KbqTag
         this.textElement.nativeElement.scrollTo?.({ left: 0, behavior: 'instant' });
     }
 
-    private dispatchSelectionChange(isUserInput = false) {
-        this.selectionChange.emit({
-            source: this,
-            isUserInput,
-            selected: this.selected
-        });
+    /**
+     * Sets the selected state of the tag.
+     *
+     * @docs-private
+     */
+    setSelectedState(isSelected: boolean, isUserInput: boolean, emitEvent: boolean): void {
```

Comment:
Предлагаю сделать похожий интерфейс на `FormControl.setValue`: 
```suggestion
    setSelectedState(isSelected: boolean, { isUserInput, emitEvent }: { isUserInput: boolean; emitEvent: boolean }): void {
```